index.html

```diff
@@ -279,6 +279,21 @@ <h3>Terminology</h3>
       </section>
 
       <section id="conformance" class="normative">
+
+        <p>
+A <dfn>conforming document</dfn> is any concrete expression of the
+data model that follows the relevant normative requirements in Section
+<a href="#data-model"></a>.
+        </p>
+
+        <p>
+A <dfn>conforming processor</dfn> is any algorithm realized
+as software and/or hardware that generates and/or consumes a
+<a>conforming document</a> according to the relevant normative statements in
+Section <a href="#algorithms"></a>. Conforming processors MUST produce errors
+when non-conforming documents are consumed.
```

Comment:
Wouldn't it be cleaner and more future-proof if this spec did not define its own terminology, but would directly refer to https://www.w3.org/TR/vc-data-model-2.0/#dfn-conforming-document, https://www.w3.org/TR/vc-data-model-2.0/#dfn-conforming-issuer-implementation, and https://www.w3.org/TR/vc-data-model-2.0/#dfn-conforming-verifier-implementation instead? This means if, at any time, there is a change in the VCDM spec in this respect, no discrepancy would occur. Besides, for example for the conforming document case, the aforementioned reference in the VCDM seems to be more precise and more detailed as what we have here...

Reply:
We can't do that because this specification has additional data model requirements on top of the base VCDM. 

We DO require that a conforming document in this specification is ALSO a conforming VCDM data model (or is used in a conforming VCDM data model). However, we ALSO have additional fields that are used in this specification that are requirements above and beyond the base VCDM data model requirements.

Note these two normative requirements, which realize the statements above:

https://github.com/w3c/vc-bitstring-status-list/pull/102/files#diff-0eb547304658805aad788d320f10bf1f292797b5e6d745a3bf617584da017051R314-R316

https://github.com/w3c/vc-bitstring-status-list/pull/102/files#diff-0eb547304658805aad788d320f10bf1f292797b5e6d745a3bf617584da017051R436-R439